### ISSUE TITLE
SocketResponseModel does not have to be a class.

### ISSUE THREAD

**Issue by FetzenRndy** [AndyGura/nestjs-socket-handlers-with-ack#5]
## The Problem

The SocketResponseModel is declared as a class, even though it does not have to.
By declaring it as an interface or a type you make sure the typescript compiler won't emit a runtime for
this "data"-class

## How to fix?

Just make it a type or a Interface:

```ts
export type SocketResponseModel = {
	success: boolean;
	msg: any;
};
```

or 

```ts
export interface SocketResponseModel {
	success: boolean;
	msg: any;
};
```

**Comment by AndyGura** [comment]
@FetzenRndy  fixed with 0f1886ecea172b64ea9d0c72ece39841860ee75c

**Comment by AndyGura** [comment]
Included in version 1.1.0
